README.md

```diff
@@ -103,3 +103,30 @@ See the [GOB-Workflow README](https://github.com/Amsterdam/GOB-Workflow/blob/dev
 See [GOB-Core start_commands.json](https://github.com/Amsterdam/GOB-Core/blob/master/gobcore/workflow/start_commands.json)
 for instructions how to start e2e_test and data_e2e_test.
 
+## Data end-2-end tests
+
+The warnings and errors that are reported are:
```

Comment:
Nice overview. Can we add the type (warning or error) for each case?

Reply:
Added